admin-center/dashboard/add-site.md

```diff
@@ -1,12 +1,16 @@
 ---
-layout: post
+layout: page
 title: How to Add a New Site
 category: manual
 tags: "help-manual settings add-site"
+date: March 2, 2017
+last_modified_at: March 2, 2017
 published: true
+sidenav: admin_center
+redirect_from: "/manual/add-site"
```

Comment:
This is the only redirect that doesn't have the url on an indented line following.

Reply:
Both seem to work, but will update!